src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -432,23 +428,9 @@
       (or (false? (:authorization-required certificate-status-access-control))
           (not-empty certificate-status-whitelist))
       (log/warn
-        "The 'client-whitelist' and 'authorization-required' settings in the"
-        "'certificate-authority.certificate-status' section are deprecated and"
-        "will be removed in a future release.  Remove these settings and create"
-        "an appropriate authorization rule in the"
-        "/etc/puppetlabs/puppetserver/conf.d/auth.conf file.")
+       (trs "The ''client-whitelist'' and ''authorization-required'' settings in the ''certificate-authority.certificate-status'' section are deprecated and will be removed in a future release.  Remove these settings and create an appropriate authorization rule in the /etc/puppetlabs/puppetserver/conf.d/auth.conf file."))
```

Comment:
So 'client-whitelist', etc. Those are not going to be translated strings, I'm guessing.  So I would suggest pulling those out into interpolated variables.  You might also pull this first sentence out as a separate string, as then it could be translated once (once for this line, and one for line 433).  You'd need to join the resulting and following sentences again, with a (str) or something, to insert the space between it and the following sentences... but from a translation standpoint, it's probably worth it.